ota-plus-web/app/com/advancedtelematic/controllers/ClientToolController.scala

```diff
@@ -112,7 +112,7 @@ class ClientToolController @Inject()(
       } else {
         throw RemoteApiError(result, s"GET root.json returns error: ${body.toString}")
       }
-    targetKeys <- keyServerApi.targetKeys(repoId)
+    targetKeys <- keyServerApi.targetKeys(repoId).recover { case _: RemoteApiParseError => Seq.empty }
```

Comment:
Better to recover if keys not found 404, rather than accept errors from TUF service.

Reply:
And also fix TUF service to return proper error 404.